.ci/jenkins/jenkins-trigger-nnpi.groovy

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
```

Comment:
Can we use the test/ci/jenkins folder to host this script?

This trigger is separate from buildkite right? this connects our github-repo <-> jenkins.
Buildkite is not involved. Is that the correct understanding

Reply:
Meant this as a general comment for all the PRs

Reply:
Moved the script to test/ci/jenkins. 
Correct, the trigger is separate from buildkite. The nnp/nnpi triggers run on Jenkins and buildkite is not involved.